app/enterprise/changelog.md

```diff
@@ -4,6 +4,223 @@ no_search: true
 no_version: true
 ---
 
+## 2.5.0.0 (beta)
```

Comment:
nit: In the event we release another beta I think it would be better to suffix with a `1`; WDYT?

```suggestion
## 2.5.0.0 (beta1)
```

Reply:
That works for me - thanks @mikefero